docs/_posts/2016-11-11-readfarm-case-forestry.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 那不是秋天的红叶——林业有害生物监测
+author: read farm
+categories: news
+tags:
+  - UAV
+  - RS
+  - spectrum
+  - forestry
+  - pine nematode
+  - Read Farm
+image: assets/2016/11-readfarm-forestry/0.png
+---
+
+森林一直被誉为地球最美的外衣，是维护生态平衡的重要保障，同时具有巨大经济价值，而我国也是全球利用造林减排的最大国。在林业众多病虫害当中，松材线虫是一种破坏性极强、漫延迅速且难处置的一种虫害，自1982年首次发现松材线虫至今，松材线虫的疫区已经扩大至15个省，致死松树达5亿多株，经济损失近千亿元。
+
+2016年观耘团队与广东省林科院开展有害生物及林业资源调查项目的合作，共同应对林业病虫害爆发的新挑战。双方合作至今，针对松材线虫病害的防治方案成果显著，共同发表了多篇论文及行业期刊文章，并逐步在行业中形成监测标准。年重点林区持续服务面积近30万亩。较传统人工监测手段提高效率约300倍。
+
```

Comment:
Can we introduce more about the project and the problem we solved? (3-5 paragraphs more)

Reply:
Alright, Haonan is on the way

Reply:
This article should be prepared ad a news article posted on a journey, which the readers I'd all the investors , entrepreneurs, and journerlists.

It should not be published with "easy contents".